torchinductor/codegen/triton.py

```diff
@@ -56,7 +56,7 @@ def constant(value, dtype):
             return 'float("inf")'
         elif value == float("-inf"):
             return 'float("-inf")'
-        return super().constant(value, dtype)
+        return OpOverrides.constant(value, dtype)
 
```

Comment:
Okay, showing my python ignorance here- why is OpOverrides preferable to super()?

Reply:
`super()` doesn't work inside a `staticmethod`.  This was triggering an error for me.